pages/events/brainspark.md

```diff
@@ -0,0 +1,16 @@
+---
+layout : base-layout.ejs
+title : Brainspark
+tags: ["event"]

```

Comment:
you can add tags like, "puzzle", "mind game" for now it doesn't matter. But you got the idea.